src/engine/renderer/tr_image.cpp

```diff
@@ -2195,6 +2176,47 @@ image_t *R_FindCubeImage( const char *imageName, imageParams_t &imageParams )
 		R_FreeCubePics( pic, i );
 	}
 
+	char baseName[ MAX_QPATH ];
+	COM_StripExtension3( buffer, baseName, sizeof( baseName ) );
+
+	const FS::PakInfo* bestPak = nullptr;
+	bool foundLoader = false;
+	for ( const cubeMapLoader_t &loader : cubeMapLoaders )
+	{
+		std::string altName = Str::Format( "%s.%s", baseName, loader.ext );
+		const FS::PakInfo* pak = FS::PakPath::LocateFile( altName );
+
+		// We found a file and its pak is better than the best pak we have
+		// this relies on how the filesystem works internally and should be moved
+		// to a more explicit interface once there is one. (FIXME)
+		if ( pak != nullptr && ( bestPak == nullptr || pak < bestPak ) )
+		{
+			bestPak = pak;
+			foundLoader = true;
+		}
+
+		if ( pak == nullptr ) {
```

Comment:
We don't want this homepath thing (yes it should be deleted from the other one too)

Reply:
@VReaperV Is it used to load cached reflection cubemaps?

Reply:
The one in `R_FindImageLoader()` is.

Reply:
Well, something is wrong, `R_FindImageLoader()` is not expected to be used to load a cached cubemap.

For now I'll keep the home loader in both, then later we can decide what is to be kept when things are not weird.

Reply:
> 🔹️ xreaperx: Well, I thought you were asking about the homepath usage in the loader?
> 🔹️ xreaperx: I added it so it would look for .ktx under homepath
> 🔹️ illwieckz: yes
> 🔹️ illwieckz: but you said only R_FindImageLoader requires to read home path
> 🔹️ xreaperx: Well yeah otherwise it doesn't find the files
> 🔹️ illwieckz: but this is a cubemap
> 🔹️ illwieckz: R_FindImageLoader is for non-cubemap files
> 🔹️ xreaperx: Yeah there's some weirdness with that

Reply:
> 🔹️ xreaperx: @illwieckz `R_FindCubeImage()` use `R_FindImageLoader()`, not `R_FindCubeImage()`
> 🔹️ xreaperx: So if that's not what it should do, then the error is in that

I say: we merge this that fixes the skybox, then later we fix the reflection cubemap code.

Reply:
Sounds reasonable to me. I don't know if there's supposed to be some other way to load images from homepath, but other functionality that I looked at (log, glsl cache) used `FS::HomePath`.

Reply:
This is not the code that's used for reflection maps. The reflection map search sets an `IF_HOMEPATH` flag instructing to search the homepath instead of the VFS. It's bad to search the homepath for every random image.

Reply:
Yes and no. `loader.ImageLoader()` is the only call that uses `IF_HOMEPATH`, but `R_FindImageLoader()`, which preceeds it, doesn't use that, and is used to know if the file exists in the first place. I suppose a proper solution might be to pass `imageParams.bits` to it and then use `IF_HOMEPATH`.

Reply:
The purpose of this PR is to fix skybox loading, other things like “attempting to remove some code and see if that breaks reflections“ later.